src/docs/source/includes/topics/_istio.md

```diff
@@ -0,0 +1,189 @@
+</aside>
+
+The output of the above command should look something like this:
+
+```text
+$ istioctl manifest apply --set profile=demo
+- Applying manifest for component Base...
+✔ Finished applying manifest for component Base.
+- Applying manifest for component EgressGateway...
+- Applying manifest for component Prometheus...
+- Applying manifest for component Pilot...
+- Applying manifest for component Tracing...
+- Applying manifest for component Citadel...
+- Applying manifest for component Injector...
+- Applying manifest for component Galley...
+- Applying manifest for component Kiali...
+- Applying manifest for component IngressGateway...
+- Applying manifest for component Policy...
+- Applying manifest for component Telemetry...
+- Applying manifest for component Grafana...
+✔ Finished applying manifest for component Galley.
+✔ Finished applying manifest for component Kiali.
+✔ Finished applying manifest for component Injector.
+✔ Finished applying manifest for component Prometheus.
+✔ Finished applying manifest for component Citadel.
+✔ Finished applying manifest for component Pilot.
+✔ Finished applying manifest for component Policy.
+✔ Finished applying manifest for component IngressGateway.
+✔ Finished applying manifest for component Tracing.
+✔ Finished applying manifest for component EgressGateway.
+✔ Finished applying manifest for component Telemetry.
+✔ Finished applying manifest for component Grafana.
+
+
+✔ Installation complete
+```
+
+You also need to run `kubectl get pods -n istio-system` and ensure all pods are in the running state (the value of the `STATUS` column for each pod should be `Running`)
+
+Before continuing with the Mushop deployment, you also need to label the `mushop` namespace in order for Istio to automatically inject the Envoy sidecar proxy next to each Mushop service.
+
+1. Create the `mushop` namespace:
+
+```shell
+kubectl create ns mushop
+```
+
+1. Label the namespace with `istio-injection=enabled`:
+
+```shell
+kubectl label namespace mushop istio-injection=enabled
+```
+
+1. Follow the instructions for <a href="#deployment">deploying Mushop</a>. 
+
+### Creating Istio resources
+
+In order to configure the traffic routing and the ingress gateway, you will need to deploy a Gateway resource and a VirtualService resource.
+
+1. Deploy a Gateway resource:
+
+```shell
+cat << EOF | kubectl apply -f -
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: gateway
+  namespace: mushop
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 80
+        name: http
+        protocol: HTTP
+      hosts:
+        - '*'
+EOF
+```
+
```

Comment:
Can you create windows versions of these commands as well? something like how they do for AKS https://docs.microsoft.com/en-us/azure/aks/servicemesh-istio-install?pivots=client-operating-system-windows#install-the-istio-components-on-aks